### PR TITLE
Update admindocs refs to Singularity community links

### PR DESCRIPTION
**Pull request by pedroalvesbatista**
### Description of the Pull Request (PR):
Updates references to point to Singularity Community admin docs.

**Before submitting a PR, make sure you have done the following:**

* Read the [Guidelines for Contributing](https://github.com/hpcng/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
* Added changes to the CHANGELOG if necessary according to the [Contribution Guidelines](https://github.com/hpcng/singularity/blob/master/CONTRIBUTING.md)
* Added tests to validate this PR and tested this PR locally with a `make testall`
* Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/hpcng/singularity/blob/master/CONTRIBUTING.md)
* Added myself as a contributor to the [Contributors File](https://github.com/hpcng/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: security.rst [hunk 1]
```diff
@@ -183,5 +183,5 @@ Security Considerations of Cloud Services:
 Security is not a check box that one can tick and forget.  It’s an ongoing process that begins with software architecture, and 
 continues all the way through to ongoing security practices.  In addition to ensuring that containers are run without elevated 
 privileges where appropriate, and that containers are produced by trusted sources, users must monitor their containers for newly 
-discovered vulnerabilities and update when necessary just as they would with any other software. Sylabs is constantly probing to 
+discovered vulnerabilities and update when necessary just as they would with any other software. Singularity Security Team is constantly probing to
```
Comment (DrDaveD) [inline security.rst:186]:
```suggestion
discovered vulnerabilities and update when necessary just as they would with any other software. The Singularity community is constantly probing to 
```

**Review by DrDaveD — CHANGES_REQUESTED** [review]
Please replace all the `user-docs/master` with `user-docs/\{userversion\}` and `admin-docs/master` with `admin-docs/\{adminversion\}`.  We do have those macros, and it would be better to point to the same version than the master branch.

**Comment by pedroalvesbatista** [comment]
> Please replace all the `user-docs/master` with `user-docs/\{userversion\}` and `admin-docs/master` with `admin-docs/\{adminversion\}`. We do have those macros, and it would be better to point to the same version than the master branch.

Alright, done !

**Review by DrDaveD — CHANGES_REQUESTED** [review]
Now it lost all the "user-docs" before the `/\{userversion\}` and the `admin-docs` before the `/\{adminversion\}`.  I also still see one case of `admin-docs/master`.  Please fix.

File: security.rst [hunk 2]
```diff
@@ -33,7 +33,7 @@ These measures allow users to interact with the host system from within the cont
 Singularity Image Format (SIF)
 ##############################
 
-Sylabs addresses container security as a continuous process. It attempts to provide container integrity throughout the distribution
+Singularity Community addresses container security as a continuous process. It attempts to provide container integrity throughout the distribution
```
Comment (DrDaveD) [inline security.rst:36]:
```suggestion
The Singularity community addresses container security as a continuous process. It attempts to provide container integrity throughout the distribution
```

**Comment by pedroalvesbatista** [comment]
> Now it lost all the "user-docs" before the `/\{userversion\}` and the `admin-docs` before the `/\{adminversion\}`. I also still see one case of `admin-docs/master`. Please fix.

Done !

**Review by DrDaveD — APPROVED** [review]

**Pull request merged**